src/audio/copier.c

```diff
@@ -569,7 +569,12 @@ static int copier_comp_trigger(struct comp_dev *dev, int cmd)
 		comp_err(dev, "failed to find dai comp");
 		return ret;
 	}
+
 	dai_cd = comp_get_drvdata(dai_copier);
+	/* dai is in another pipeline and it is not prepared or active */
+	if (dai_copier->state <= COMP_STATE_READY || dai_cd->endpoint->state <= COMP_STATE_READY)
+		return ret;
```

Comment:
If we are returning an error we should have some trace to help with debug,

Reply:
sure

Reply:
@RanderWang @lgirdwood we're in this situation because we propogate triggers across pipelines isnt it? Should we think about not doing it? And if we dont want to change that logic, should we return PPL_STATUS_PATH_STOP here?

Reply:
sure, dai and host are in different pipelines.  We need to do update dai position for windows driver. if we use PPL_STATUS_PATH_STOP, we should return it when just entering copier_copy without doing more, but now we finish copy but without updating position

Reply:
@ranj063  could you help to review it again ?